packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -23,26 +23,28 @@ const Duration _kDropdownMenuDuration = Duration(milliseconds: 300);
 const double _kMenuItemHeight = 48.0;
 const double _kDenseButtonHeight = 24.0;
 const EdgeInsets _kMenuItemPadding = EdgeInsets.symmetric(horizontal: 16.0);
-const EdgeInsetsGeometry _kAlignedButtonPadding = EdgeInsetsDirectional.only(start: 16.0, end: 4.0);
+const EdgeInsetsGeometry _kAlignedButtonPadding =
```

Comment:
Revert this to a single line

Reply:
Done. I have already added dart docs for _isExpanded_ and _isDense_. Since DropdownButtonFormField is actually wrapping DropdownButton, so the test for DropdownButton should cover DropdownButtonFormField too as all those newly added properties are actually passed into inner DropdownButton.